### ISSUE TITLE
translation in different language

### ISSUE THREAD

**Issue by preliot** [optimistdigital/nova-multiselect-field#16]
Translated all strings to Dutch and put tranlated strings in:

`\resources\lang\vendor\nova-multiselect\nl.json`

This doesn't work, i must place it in en.json, which is not logical.

Or am i missing something?

Tranlation file:

```php
{
  "novaMultiselect.maxElements": "Selectie van :max item(s) toegestaan.",
  "novaMultiselect.noResult": "Geen resultaten gevonden.",
  "novaMultiselect.noOptions": "Lijst is leeg.",
  "novaMultiselect.limitText": "en :count meer",
  "novaMultiselect.selectLabel": "Druk enter om te selecteren",
  "novaMultiselect.selectGroupLabel": "Druk enter om groep te selecteren",
  "novaMultiselect.selectedLabel": "Geselecteerd",
  "novaMultiselect.deselectLabel": "Druk enter om te verwijderen",
  "novaMultiselect.deselectGroupLabel": "Druk enter om groep te verwijderen",
  "novaMultiselect.placeholder": "Selecteer optie(s)",
  "novaMultiselect.nItemsSelected": ":count items geselecteerd"
}
```

**Comment by Tarpsvo** [comment]
Hey there!

Thanks for reporting this, the logic to load translation files was missing indeed. I released a new version, 1.3.5 which should address this issue. Give it a try and let me know if your `nl.json` is now loaded properly.

Thanks!

**Comment by preliot** [comment]
Checked 1.3.5 and it's fixed!
